packages/core/admin/admin/src/pages/ProfilePage/utils/api.js

```diff
@@ -16,4 +16,16 @@ const putUser = async (body) => {
   return { ...data.data, currentTheme: body.currentTheme };
 };
 
-export { fetchUser, putUser };
+const fetchSSOInfo = async () => {
```

Comment:
this seems like we're just doing it because someone else did it this way, imo there's no value to not having it inline.

Reply:
removed